tests/v2/test_peer_manager.nim

```diff
@@ -413,37 +416,44 @@ procSuite "Peer Manager":
 
     let
       node = newTestWakuNode(generateSecp256k1Key(), ValidIpAddress.init("0.0.0.0"), Port(0))
-      peer1 = parseRemotePeerInfo("/ip4/0.0.0.0/tcp/30300/p2p/" & basePeerId & "1")
-      peer2 = parseRemotePeerInfo("/ip4/0.0.0.0/tcp/30301/p2p/" & basePeerId & "2")
-      peer3 = parseRemotePeerInfo("/ip4/0.0.0.0/tcp/30302/p2p/" & basePeerId & "3")
-      peer4 = parseRemotePeerInfo("/ip4/0.0.0.0/tcp/30303/p2p/" & basePeerId & "4")
-      peer5 = parseRemotePeerInfo("/ip4/0.0.0.0/tcp/30303/p2p/" & basePeerId & "5")
+      peer1 = parsePeerInfo("/ip4/0.0.0.0/tcp/30300/p2p/" & basePeerId & "1")
+      peer2 = parsePeerInfo("/ip4/0.0.0.0/tcp/30301/p2p/" & basePeerId & "2")
+      peer3 = parsePeerInfo("/ip4/0.0.0.0/tcp/30302/p2p/" & basePeerId & "3")
+      peer4 = parsePeerInfo("/ip4/0.0.0.0/tcp/30303/p2p/" & basePeerId & "4")
+      peer5 = parsePeerInfo("/ip4/0.0.0.0/tcp/30303/p2p/" & basePeerId & "5")
+
+    require:
+      peer1.isOk()
+      peer2.isOk()
+      peer3.isOk()
+      peer4.isOk()
+      peer5.isOk()
 
     # service peers
-    node.peerManager.addServicePeer(peer1, WakuStoreCodec)
-    node.peerManager.addServicePeer(peer2, WakuFilterCodec)
-    node.peerManager.addServicePeer(peer3, WakuLightPushCodec)
-    node.peerManager.addServicePeer(peer4, WakuPeerExchangeCodec)
+    node.peerManager.addServicePeer(peer1.value, WakuStoreCodec)
+    node.peerManager.addServicePeer(peer2.value, WakuFilterCodec)
+    node.peerManager.addServicePeer(peer3.value, WakuLightPushCodec)
+    node.peerManager.addServicePeer(peer4.value, WakuPeerExchangeCodec)
 
     # relay peers (should not be added)
-    node.peerManager.addServicePeer(peer5, WakuRelayCodec)
+    node.peerManager.addServicePeer(peer5.value, WakuRelayCodec)
 
     # all peers are stored in the peerstore
     check:
-      node.peerManager.peerStore.peers().anyIt(it.peerId == peer1.peerId)
-      node.peerManager.peerStore.peers().anyIt(it.peerId == peer2.peerId)
-      node.peerManager.peerStore.peers().anyIt(it.peerId == peer3.peerId)
-      node.peerManager.peerStore.peers().anyIt(it.peerId == peer4.peerId)
+      node.peerManager.peerStore.peers().anyIt(it.peerId == peer1.value.peerId)
```

Comment:
see my other comments for all this file. instead of having `.value` perhaps use `xxRes`